io.github.shiiion.primehack.yml

```diff
@@ -102,10 +102,11 @@ modules:
     cleanup:
       - /share/man
     post-install:
-      - install -D dolphin-emu-wrapper /app/bin/dolphin-emu-wrapper
-      - install -Dm644 appdata.xml /app/share/appdata/dolphin-emu.appdata.xml
-      - sed -i -e 's/"2048"/"512"/g' /app/share/icons/hicolor/scalable/apps/dolphin-emu.svg
-      - desktop-file-edit --set-key=Exec --set-value='dolphin-emu-wrapper' /app/share/applications/dolphin-emu.desktop
+      - install -D primehack-wrapper /app/bin/primehack-wrapper
+      - install -Dm644 appdata.xml /app/share/appdata/primehack.appdata.xml
+      - rm /app/share/icons/hicolor/scalable/apps/dolphin-emu.svg
+      - desktop-file-edit --set-key=Exec --set-value='primehack-wrapper' /app/share/applications/dolphin-emu.desktop
+      - desktop-file-edit --set-key=Name --set-value='PrimeHack' /app/share/applications/dolphin-emu.desktop
```

Comment:
I think you still need to change the `dolphin-emu.desktop`s, or it will still collide.

Reply:
I believe this is handled by the `rename-desktop-file` entry which renames it to `io.github.shiiion.primehack.desktop`

Reply:
I'll update it to move the file to 'primehack.desktop'. It's less confusing that way.